data/movements/scripts/tiles.lua

```diff
@@ -7,7 +7,7 @@ function onStepIn(creature, item, position, fromPosition)
 	end
 
 	local player = creature:getPlayer()
-	if player == nil or player:isInGhostMode() then
+	if not player player or player:isInGhostMode() then
```

Comment:
Ye expected to get issues with this pr, forgot to write "double check it"
Ill fix in a couple of mins :p